tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source.ts

```diff
@@ -29,14 +29,13 @@ import {
 } from './tb_feature_flag_data_source_types';
 
 const DARK_MODE_MEDIA_QUERY = '(prefers-color-scheme: dark)';
+const FEATURE_FLAG_STORAGE_KEY = 'tb_feature_flag_storage_key';
 
 // TODO(tensorboard-team): QueryParamsFeatureFlagDataSource now is a misnomer as
```

Comment:
I think you can remove this comment now that you've renamed it. I think we've already figured out the composability aspect, too.